error/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub enum Error_type {
     PSM_OK_NO_PROGRESS = 1,
     PSM_PARAM_ERR = 3,
     PSM_NO_MEMORY = 4,
-    PSM_INIT_NOT_INIT = 5, 
+    PSM_INIT_NOT_INIT = 5,
```

Comment:
Ooh, weird issue 2. Do we need explicit conversions here for backwards compatibility.